### PR TITLE
Rating CRUD

### PR DESCRIPTION
**Pull request by Qod1rc1k**
rating CRUD

### CODE REVIEW

File: src/main/java/com/example/onefit/rating/RatingController.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<RatingResponseDto> update(@PathVariable UUID id, @RequestBody RatingUpdateDto ratingUpdateDto){
+        RatingResponseDto ratingResponseDto = ratingService.update(ratingUpdateDto, id);
+        return ResponseEntity.ok(ratingResponseDto);
+    }
+
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable UUID id){
```
Comment (mavjudbekdev) [inline src/main/java/com/example/onefit/rating/RatingController.java]: deleteda Response Notogri qaytarilgan

File: src/main/java/com/example/onefit/rating/RatingController.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public ResponseEntity<RatingResponseDto> get(@PathVariable UUID id){
+        RatingResponseDto ratingResponseDto = ratingService.get(id);
+        return ResponseEntity.ok(ratingResponseDto);
+    }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<RatingResponseDto> update(@PathVariable UUID id, @RequestBody RatingUpdateDto ratingUpdateDto){
+        RatingResponseDto ratingResponseDto = ratingService.update(ratingUpdateDto, id);
+        return ResponseEntity.ok(ratingResponseDto);
+    }
```
Comment (mavjudbekdev) [inline src/main/java/com/example/onefit/rating/RatingController.java:43]: update kerk emas

File: src/main/java/com/example/onefit/rating/dto/RatingUpdateDto.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.example.onefit.rating.dto;
+
+public class RatingUpdateDto {
```
Comment (mavjudbekdev) [inline src/main/java/com/example/onefit/rating/dto/RatingUpdateDto.java:3]: bu ham kerk emas

**Review by mavjudbekdev — CHANGES_REQUESTED** [review]

**Review by mavjudbekdev — APPROVED** [review]

**Comment by Qod1rc1k** [comment]
Rating CRUD

**Pull request merged**